utils/utils_epd.py

```diff
@@ -0,0 +1,146 @@
+# This file is released under MIT licence, see file LICENSE.
+# Author(s):       Theo Lacombe
+#
+# Copyright (C) 2021 Inria
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.patches import Polygon
+import gudhi as gd
+
+def sample_circle(n):
```

Comment:
We have a function like this in master, but indeed not released yet, so you still need yours.

Reply:
Yes, I will update the notebook later to account for this I think.